internal/migrations/observability/api/pipelines_test.go

```diff
@@ -51,12 +51,12 @@ var _ = Describe("#ConvertPipelines", func() {
 			Labels: map[string]string{"foo": "bar"},
 		}, []obs.FilterSpec{
 			{
-				Name:            "filter-labelPipeline-openShiftLabels",
+				Name:            "filter-labelPipeline-" + openshiftLabelsFilterName,
```

Comment:
The regex is saying no capital letters am I right? The `P` should be lowercase.
```suggestion
				Name:            "filter-labelpipeline-" + openshiftLabelsFilterName,
```